Website/src/components/Headers/HomeHeader.jsx

```diff
@@ -33,14 +33,10 @@ function IndexHeader() {
           ref={pageHeader}
         ></div>
         <Container>
-          <div className="content-center brand">
-            <img
-              alt="..."
-              className="n-logo"
-              src={require("assets/img/tkot/tkot-logo-512x512.png")}
-            ></img>
-            <h1 className="h1-seo">Te Kahu o Taonui</h1>
-            <h3>Nau mai, haere mai</h3>
+          <div className="float-left" style={{paddingLeft: '10px', paddingTop: '200px'}}>
+            
+            <h1 className="h1-seo" style={{fontFamily: 'Roboto'}}>Te Kahu o Taonui</h1>
+            <h3 style={{color: '#8f95c3', textAlign: 'left'}}>Nau mai, haere mai</h3>
```

Comment:
Same previous comment. Please move all styles to the tkot.scss file under an appropriate CSS class name, then reference the CSS class name in the component.